app/konnect/servicehub/manage-services.md

```diff
@@ -61,6 +61,61 @@ box to save.
 
 4. Enter a version name and click **Create** to save.
 
+    Service versions have a Published status by default upon creation.
+
+### Set a Service Version to Draft
+
+1. From the left navigation menu, click **Services**.
+
+2. Select a Service from the dropdown menu.
+
+3. Navigate to **Versions**.
+
+4. Click on the version you want to set as a draft, and you'll be taken to the version detail page.
+
+5. Click on the **Actions** dropdown menu.
+
+6. Click **Set Version as Draft** to set the Service version as a draft. The service version will not appear
+   on the Dev Portal UI and will still be editable in the Konnect Admin UI.
+
+### Archive a Service Version
+
+1. From the left navigation menu, click **Services**.
+
+2. Select a Service from the dropdown menu.
+
+3. Navigate to **Versions**.
+
+4. Click on the version you want to archive, and you'll be taken to the version detail page.
+
+5. Click on the **Actions** dropdown menu.
+
+6. Click **Archive Version** to archive the Service version. Archiving prevents that 
+version from showing up on the Dev Portal UI.
+
+    The Service version will still appear on the Konnect Admin UI, however users 
+    will not be able to send requests to the Service version endpoints, and those 
+    endpoints will no longer be supported.
```

Comment:
Does this mean that this functionality isn't just for Dev Portal? Or is this referring to applications registered to the service version through the Dev Portal?

Reply:
Yes, from my discussions with engineering, the endpoints are no longer supported when the service version is archived.

Reply:
The phrase "...will still appear on the Konnect Admin UI" can be more explicit where exactly we mean. Can we say "The Service version will still appear when managing services through Service Hub, however..."